ooyala_player/ooyala_player.py

```diff
@@ -360,13 +383,7 @@ class OoyalaPlayerBlock(OoyalaPlayerMixin, XBlock):
 
     @property
     def api_key_3play_with_default_setting(self):
-        if self.api_key_3play:
-            return self.api_key_3play
-
-        settings_service = self.runtime.service(self, 'settings')
-        if settings_service:
-            return settings_service.get_settings_bucket(self).get('3PLAY_API_KEY')
-        return None
+        return self.get_attribute_or_default('api_key_3play', '3PLAY_API_KEY')
```

Comment:
`get_attribute_or_default` does not take the second parameter anymore. I don't think we need the `api_key_3play_with_default_setting` method/property at all now, though Can we just remove it completely?